apis/kueue/v1beta1/provisioningrequestconfig_types.go

```diff
@@ -21,10 +21,15 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+type ProvisioningRequestConfigPodSetMergePolicy string
+
 const (
 	// ProvisioningRequestControllerName is the name used by the Provisioning
 	// Request admission check controller.
 	ProvisioningRequestControllerName = "kueue.x-k8s.io/provisioning-request"
+
+	IdenticalSchedulingRequirements ProvisioningRequestConfigPodSetMergePolicy = "IdenticalSchedulingRequirements"
```

Comment:
rename to IdenticalWorkloadSchedulingRequirements as per:
https://github.com/kubernetes-sigs/kueue/pull/5290#discussion_r2100227988